tools/build_third_party_libs.py

```diff
@@ -51,6 +51,84 @@
     elif sys.platform == "linux":
         platform_dir = "Linux"
 
+    #
+    # Boost
+    #
+
+    print("[SPEAR | build_third_party_libs.py] Building Boost...")
+
+    boost_dir = os.path.realpath(os.path.join(third_party_dir, "boost"))
+    include_dir = os.path.realpath(os.path.join(third_party_dir, "boost", "boost"))
+    build_dir = os.path.realpath(os.path.join(third_party_dir, "boost", "stage"))
```

Comment:
Let's vertically align these statements.

Reply:
I agree with you, although this is a matter of taste, and our current convention does not mandate that we align equal signs in all cases. But in this case, it clearly makes the code more beautiful.